src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -7,9 +7,7 @@
 
     /* Prefix definitions */
     public static final Prefix PREFIX_NAME = new Prefix("n/");
+    public static final Prefix PREFIX_NRIC = new Prefix("ic/");
     public static final Prefix PREFIX_PHONE = new Prefix("p/");
```

Comment:
Just a point for discussion, should our prefix be '/c' for Contact, or '/p' for Phone?